samcli/lib/telemetry/event.py

```diff
@@ -21,11 +21,27 @@ class EventName(Enum):
     USED_FEATURE = "UsedFeature"
     DEPLOY = "Deploy"
     BUILD_FUNCTION_RUNTIME = "BuildFunctionRuntime"
+    SYNC_USED = "SyncUsed"
+    SYNC_FLOW_START = "SyncFlowStart"
+    SYNC_FLOW_END = "SyncFlowEnd"
 
 
 class EventType:
     """Class for Events and the types of values they may have."""
 
+    _SYNC_FLOWS = [
+        "AliasVersionSyncFlow",
```

Comment:
If we're including `FunctionSyncFlow` here as the parent of Zip and Image functions, should we also include `GenericApiSyncFlow`? I'm not certain on how these definitions are used downstream in the metrics.

Reply:
My logic was that, since `GenericApiSyncFlow` is described as an interface for `HttpApiSyncFlow` and `RestApiSyncFlow`, it wouldn’t exist unless something were to go wrong. Though, that was also the reason I included `FunctionSyncFlow`. Adding in the extra value is definitely possible, I’ll get on that as soon as I’ve got a minute!